cypress/integration/notes_spec.js

```diff
@@ -0,0 +1,31 @@
+describe("Server set up", () => {
+  beforeEach(() => {
+    cy.exec("yarn run db:drop");
+  });
```

Comment:
Should this be run once for a test suite rather than in every spec? Spec helper type thing.